tests/tests_integration/test_rules/test_exporters/test_dms_exporters.py

```diff
@@ -49,3 +150,72 @@ def test_export_to_cdf(self, cognite_client: CogniteClient, alice_rules: DMSRule
 
         assert uploaded_by_name["spaces"].total == 1
         assert uploaded_by_name["spaces"].failed == 0
+
+    def test_export_pipeline_populate_and_retrieve_data(
```

Comment:
This is an intentional long test to test the full flow of creating a data model, exporting it,
populating the raw table, run the transformations, and verify that the data has been populated in the data model.